src/framework/audio/audiomodule.cpp

```diff
@@ -61,11 +63,14 @@ using namespace muse::audio::fx;
 #endif
 
 #ifdef Q_OS_LINUX
-#include "internal/platform/lin/linuxaudiodriver.h"
+#include "internal/platform/lin/alsaaudiodriver.h"
+#ifdef MUSE_PIPEWIRE_AUDIO_DRIVER
+#include "internal/platform/lin/pwaudiodriver.h"
+#endif
```

Comment:
For consistency with the CMake code, shouldn't these lines be there fore BSD too?

Reply:
They probably should. 
Pipewire officially only supports Linux, but there is a [port for FreeBSD](https://www.freshports.org/multimedia/pipewire).
I'll make the option to OFF by default on BSD.